app/graphql/resolvers/base_resolver.rb

```diff
@@ -37,7 +37,9 @@ def admin?
   end
 
   def matching_user(submission, session_id)
-    submission.user&.gravity_user_id == @context&.[](:current_user) ||
-      submission.session_id == session_id
+    (
+      submission.user.present? &&
```

Comment:
We removed `validates :gravity_user_id, presence: true` from user model -> a user can exist without this field
```suggestion
      submission.user&.gravity_user_id.present? &&
```